pkgs/development/python-modules/scspell/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  nix-update-script,
+  pyxdg,
+  setuptools,
+  writableTmpDirAsHomeHook,
+}:
+
+buildPythonPackage rec {
+  pname = "scspell";
+  version = "2.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "myint";
+    repo = "scspell";
+    tag = "v${version}";
+    hash = "sha256-XiUdz+uHOJlqo+TWd1V/PvzkGJ2kPXzJJSe5Smfdgec=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pyxdg
+  ];
+
+  nativeCheckInputs = [ writableTmpDirAsHomeHook ];
+
+  passthru.tests = {
+    version = {
+      command = [
+        "scspell"
+        "--version"
+      ];
+    };
+    help = {
+      command = [
+        "scspell"
+        "--help"
+      ];
```

Comment:
Not sure this is needed? As `passthru` is not run when building the package, this doesn't really do anything for you.